src/main.rs

```diff
@@ -521,6 +521,9 @@ impl Inlyne {
                                                         &self.opts.file_path,
                                                         contents,
                                                     );
+                                                    // TODO: Once and if history is implemented,
+                                                    // old scroll_y might be stored there
+                                                    self.renderer.set_scroll_y(0.);
                                                 }
```

Comment:
Note I didn't call `window.request_redraw()` as in `scroll_pixels`. It worked without it for me.
Furthermore, when adding the call it blinked, there is a few frames where I can see the original md at the top and then it switches.

My guess is the window is redrawn by the above call `watcher.update_file` once md is interpreted.